node/src/server-modules/GlideJson.ts

```diff
@@ -404,6 +404,58 @@ export class GlideJson {
         return _executeCommand<ReturnTypeJson<number>>(client, args);
     }
 
+    /**
+     * Searches for the first occurrence of a `scalar` JSON value in the arrays at the `path`.
+     * Out of range errors are treated by rounding the index to the array's `start` and `end.
+     * If `start` > `end`, return -1 (not found).
+     *
+     * @param client - The client to execute the command.
+     * @param key - The key of the JSON document.
+     * @param path - The path within the JSON document.
+     * @param scalar - The scalar value to search for.
+     * @param options - (Optional) Additional parameters:
+     * - (Optional) `start`: The start index, inclusive. Default to 0 if not provided.
+     * - (Optional) `end`: The end index, exclusive. Default to 0 if not provided.
```

Comment:
I think default there is -1

Reply:
Here is the original wording in AWS documentation:
end (optional) – The end index, exclusive. Defaults to 0 if not provided, which means that the last element is included. 0 or -1 means the last element is included.

I think I will update to include 0 or -1 and add explanations.

Reply:
oh
Or maybe skip this sentence at all. Keep in mind that those docs could be inaccurate as well.

Reply:
How about this change?
```suggestion
     * - (Optional) `end`: The end index, exclusive. Default to 0 if not provided.
     *                                  0 or -1 means the last element is included.
```

Reply:
Was this tested? What's the actual behaviour?

Reply:
I have the tests in IT to verify 0 and -1 for `end` index.